iodhbwm.tex

```diff
@@ -261,13 +261,16 @@
                 Eine ausführliche Beschreibung befindet sich auf \url{https://tex.stackexchange.com/a/69484/142408}.
             
             \subsubsection{TeXlive}
-                \Warning{Die Installationsanweisung für TeXlive wurde nicht getestet!}
+                \Info{Manche dieser Schritte können villeicht übersprungen werden!}
                 
                 \begin{enumerate}
                     \item \verb|kpsewhich -var-value TEXMFHOME| zum Auffinden des Ordner
-                    \item ~/texmf/tex/latex/ (ggf. neu anlegen)
-                    \item  Ordner iodhbwm nach kopieren ~/texmf/tex/latex/ (Ordner muss zwingend iodhbwm heißen)
+                    \item ~/.texmf-config/tex/latex/ (ggf. neu anlegen)
+                    \item Ordner iodhbwm nach ~/.texmf-config/tex/latex/ kopieren (Ordner muss zwingend iodhbwm heißen)
+                    \item kpsepath -n latex tex ausführen
+                    \item mktexlsr ~/.texmf-config ausführen
                     \item texhash ~/texmf ausführen
```

Comment:
Die Zeile müsste noch gelöscht werden oder nicht?

Reply:
texhash is actually calling mktexlsr. So `texhash ~/.texmf-config` should do the work.

Reply:
Yes, I meant that this way the instruction is basically there twice and one could be removed :)